src/workloads/query/TimeseriesTSBSQuery.yml

```diff
@@ -0,0 +1,1188 @@
+    Phases:
+      OnlyActiveInPhases:
+        Active: [16]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationName: aggregate
+              OperationCommand:
+                Pipeline:
+                  # All usage_* fields are random integers between 0-100.
+                  [
+                    {
+                      $match:
+                        {
+                          $or:
+                            [
+                              { usage_guest: { $gte: 25, $lt: 75 } },
+                              { usage_nice: { $gt: 50 } },
+                            ],
+                        },
+                    },
+                    {
+                      $group:
+                        {
+                          _id: { $dateTrunc: { date: "$time", unit: "hour" } },
+                          min_usage_softirq: { $min: "$usage_softirq" },
+                          min__usage_steal: { $min: "$usage_steal" },
+                          min_usage_guest: { $min: "$usage_guest" },
+                          min_usage_user: { $min: "$usage_user" },
+                          min_usage_system: { $min: "$usage_system" },
+                        },
+                    },
+                    { $sort: { _id: 1 } },
+                  ]
+
+  - Name: match_on_measurements_selective_query
+    Type: CrudActor
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [17]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationName: aggregate
+              OperationCommand:
+                Pipeline:
+                  # All usage_* fields are random integers between 0-100.
+                  [
+                    {
+                      $match:
+                        {
+                          $or:
+                            [
+                              { usage_guest: { $gte: 12, $lt: 30 } },
+                              { usage_nice: { $gt: 50, $lte: 60 } },
+                            ],
+                        },
+                    },
+                    {
+                      $group:
+                        {
+                          _id: { $dateTrunc: { date: "$time", unit: "hour" } },
+                          min_usage_softirq: { $min: "$usage_softirq" },
+                          min__usage_steal: { $min: "$usage_steal" },
+                          min_usage_guest: { $min: "$usage_guest" },
+                          min_usage_user: { $min: "$usage_user" },
+                          min_usage_system: { $min: "$usage_system" },
+                        },
+                    },
+                    { $sort: { _id: 1 } },
+                  ]
+
+  - Name: match_on_measurement_meta_time
```

Comment:
What optimization does this pipeline target? Would it use index on the metaField or a collscan?